gui/src/app/state/transactions.rs

```diff
@@ -126,8 +127,24 @@ impl State for TransactionsPanel {
                                 .to_sat()
                                 .checked_div(tx.tx.vsize().try_into().unwrap())
                                 .unwrap();
-                            let modal =
-                                CreateRbfModal::new(tx.tx.txid(), is_cancel, prev_feerate_vb);
+                            let descendant_txids: HashSet<_> = cache
```

Comment:
I would like we do not use the cache coins for this usage. cache is generally used to display something like the previous balance, while the new balance is calculated in order to not display zero values.

I think it would be good instead, to have CreateRbfModal::new() -> (modal, command)
and give the command result to modal.update so it updates its conflicting txids.

What do you think ?

Reply:
That sounds good thanks, I'll make this change. After we implement https://github.com/wizardsardine/liana/issues/677, then we'll be able to retrieve just the spending coins in this call.

Reply:
I've now made the change to use `daemon.list_coins` and pass the result to another message.

In the end, I didn't pass the result to `modal.update` because when I tried this, the modal would open without any warnings and then update shortly afterwards, and I felt it was better that the warnings would be there as soon as the modal opened. So the modal is created using the result of `list_coins`.

I made a similar change for the set of conflicting transactions when broadcasting a transaction.